### PR TITLE
Yy feature createreadme

### PR DESCRIPTION
**Pull request by yenyin262**
fixed markdown

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,58 +3,53 @@
...
+### Store Screen  
  
 ![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 31](https://user-images.githubusercontent.com/43800526/64897369-06140a00-d638-11e9-8e65-22e07a1d391b.png)
 
 
-### Brand Screen 
+### New Arrivals 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 34](https://user-images.githubusercontent.com/43800526/64897341-f0064980-d637-11e9-88eb-cbdbe8dfa014.png)
+![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
```
Comment (siddharthparmar7) [inline README.md]: Please provide a gif file that shows different screens from your app.

File: README.md [hunk 2]
```diff
@@ -3,58 +3,53 @@
...
 
-### Sale 
+```Git Clone the repository``` 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 50](https://user-images.githubusercontent.com/43800526/64897252-9ef65580-d637-11e9-8594-9b9c06664449.png)
+```yarn install``` 
 
+```cd ios``` 
 
+```pod install```
```
Comment (siddharthparmar7) [inline README.md:21]: Please provide detailed instructions on how to run your app.
cd into client dir
yarn install
cd into ios
pod install
go back to client dir
run your app.

File: README.md [hunk 3]
```diff
@@ -3,58 +3,53 @@
...
+```react-native run-ios (react-native run-ios)```
 
-# Instructions to run the project 
+# Technologies: 
 
-1. Git Clone the repository 
+```GraphCool```
 
-1. yarn install 
+```GraphQL```
```
Comment (siddharthparmar7) [inline README.md]: Do not need to add GraphQL as a separate technology. GraphCool is using GraphQL.

File: README.md [hunk 4]
```diff
@@ -3,58 +3,53 @@
...
 
-1. cd ios 
+```	React Native```
```
Comment (siddharthparmar7) [inline README.md]: why is that a huge space before the word react native?

File: README.md [hunk 5]
```diff
@@ -3,58 +3,53 @@
...
 
-1. pod install
+```Async Storage```
 
-1. react-native run-ios (react-native run-ios)
+```React-Native Maps```
 
-# Technologies: 
+```Apollo-Client```
```
Comment (siddharthparmar7) [inline README.md]: just write Apollo.

File: README.md [hunk 6]
```diff
@@ -3,58 +3,53 @@
...
+```Apollo-Client``` 
 
-1.	GraphCool 
-1.	GraphQL
-1.	React Native
-1.	Async Storage
-1.	React-Native Maps 
-1.	Apollo-Client 
-1.	React-Native Tab-Views 
+```React-Native Tab-Views```
```
Comment (siddharthparmar7) [inline README.md]: No need to include the packages you installed.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Nice work! Please make the changes. Thank you.

File: README.md [hunk 7]
```diff
@@ -1,60 +1,51 @@
...
-### New Arrivals 
+```Git Clone the repository``` 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 46](https://user-images.githubusercontent.com/43800526/64897309-d1a04e00-d637-11e9-8e3f-e0faa4190614.png)
+```cd into client dir```
 
+```yarn install``` 
 
-### Sale 
+```cd into ios```
```
Comment (siddharthparmar7) [inline README.md]: Write the actual commands, not pseudo-code. It was just to show you the steps.

File: README.md [hunk 8]
```diff
@@ -1,60 +1,51 @@
...
+```GraphCool```
 
-1. yarn install 
+```React Native```
 
-1. cd ios 
+```Async Storage```
 
-1. pod install
+```React-Native Maps```
```
Comment (siddharthparmar7) [inline README.md]: it's a library. Please give info if you are using Apple maps or Google maps

File: README.md [hunk 9]
```diff
@@ -1,60 +1,51 @@
...
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 50](https://user-images.githubusercontent.com/43800526/64897252-9ef65580-d637-11e9-8594-9b9c06664449.png)
+```pod install```
 
+```go back to client dir```
 
+run your app  with  ```react-native run-ios ```
 
 
-# Instructions to run the project 
+# Technologies:
```
Comment (siddharthparmar7) [inline README.md]: It should not be an H1 title. Use the same level of heading as other sections in your README.

File: README.md [hunk 10]
```diff
@@ -1,60 +1,51 @@
...
  
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 31](https://user-images.githubusercontent.com/43800526/64897369-06140a00-d638-11e9-8e65-22e07a1d391b.png)
-
+![Alt Text] (https://media.giphy.com/media/YPtCW7ow8HpTiisAPP/giphy.gif)
 
-### Brand Screen 
 
-![Simulator Screen Shot - iPhone X - 2019-09-13 at 13 43 34](https://user-images.githubusercontent.com/43800526/64897341-f0064980-d637-11e9-88eb-cbdbe8dfa014.png)
 
+# Instructions to run the project
```
Comment (siddharthparmar7) [inline README.md]: This should be an H1 heading. Please change it.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Review by piahmed — APPROVED** [review]
Looks good :+1:

**Comment by piahmed** [comment]
Hey @siddharthparmar7 I'm merging this as @yenyin262 is a bit stressed and was looking to merge. If you think this requires additional changes, please let her know. Thanks!

**Pull request merged**